modules/oxide-icons-default/package.json

```diff
@@ -25,13 +25,6 @@
   "author": "Ephox Corporation DBA Tiny Technologies, Inc",
   "license": "GPL-2.0-or-later",
   "devDependencies": {
-    "gulp": "^4.0.2",
-    "gulp-clean": "^0.4.0",
-    "gulp-clean-css": "^4.3.0",
-    "gulp-connect": "^5.7.0",
-    "gulp-less": "^5.0.0",
-    "gulp-rename": "^2.0.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-stylelint": "^13.0.0"
+    "grunt": "^1.6.1"
```

Comment:
TinyMCE puts all dev dependencies used by multiple projects at the monorepo root
```suggestion
```